forest/cli/src/lib.rs

```diff
@@ -1,3 +1,584 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 pub mod cli;
+pub mod subcommand;
+
+pub mod humantoken {
```

Comment:
How about moving it into a separate file? If you need to declare a new module it's a good indice of that.